modAionImpl/src/org/aion/zero/impl/db/AionBlockStore.java

```diff
@@ -953,6 +953,58 @@ public String dumpPastBlocks(long numberOfBlocks, String reportsFolder) throws I
         }
     }
 
+    public String dumpPastBlocksForConsensusTest(long firstBlock, String reportsFolder)
+            throws IOException {
+        lock.readLock().lock();
+
+        try {
+            if (firstBlock < 0) {
+                return null;
+            }
+            long lastBlock = firstBlock - 3;
```

Comment:
probably give a commit what the 3 blocks mean?

Reply:
done